qa/workunits/suites/mgr_mod_snap_schedule.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/bash
+TESTDIR=$(dirname $0)
+# run for 16 minutes
+RUN_FOR=16
+EXPECTED_SNAPS=15
+RNG_SEED=${RNG_SEED:-$(date +%s)}
+
+echo "TESTDIR=$TESTDIR"
+echo "RUN_FOR=$RUN_FOR"
+echo "EXPECTED_SNAPS=$EXPECTED_SNAPS"
+echo "RNG_SEED=$RNG_SEED"
+echo "PYTHON=$(which python3)"
+
+ceph mgr module enable snap_schedule
+sleep 2
+ceph config set mgr mgr/snap_schedule/allow_m_granularity True
+sleep 2
+mkdir z
+pushd z
+fs_path=$(pwd | sed -e "s|^$CEPH_MNT||g")
+ceph fs snap-schedule add $fs_path 1M
+
+(python3 -c "import os
```

Comment:
Does this need a workload generator? Ideally, this should run as a post workload verification that checks if enough snapshots have been taken during the course of the run.

Reply:
Coming back to this - ideally, the mgr thrasher should run alongside workloads after a snap schedule is set. Post workload completion, there needs to be checks to verify if desired number of snapshots are taken. @mchangir @batrick